compiler/test/stdlib/array.test.gr

```diff
@@ -1,7 +1,10 @@
 module ArrayTest
 
+include "list"
 include "array"
+from Array use { Immutable as ImmArray }
```

Comment:
I kinda like what @phated did to test some other submodules. He made a submodule in the test file to test the submodule, and did all of the assertions in there. What do you think about following that pattern here?

Reply:
Sure, that sounds good